### PR TITLE
Fix workshop flow after rearranging 

### PR DESCRIPTION
**Pull request by MahrRah**

### CODE REVIEW

File: 04-custom-metrics/README.md [hunk 1]
```diff
@@ -161,11 +162,13 @@ make deploy
 
 </details>
 
-## 🖼️ Visualizing custom metrics
+## 👀 Find newly created custom metrics
```
Comment (MaggieSalak) [inline 04-custom-metrics/README.md:165]: Maybe just: 

```suggestion
## 👀 Find your custom metrics
```

or 'Find your created custom metrics'?
I think we don't need to add that they are newly created in the title

File: 04-custom-metrics/README.md [hunk 2]
```diff
@@ -161,11 +162,13 @@ make deploy
...
+
+Now that we defined our custom metrics, let's find the im the Azure Portal and see the fruits of our work!
```
Comment (MaggieSalak) [inline 04-custom-metrics/README.md:167]:
```suggestion
Now that we defined our custom metrics, let's find them in the Azure Portal and see the fruits of our work!
```

File: 04-custom-metrics/README.md [hunk 3]
```diff
@@ -174,7 +177,8 @@ Go to Application Insights and select the **Metrics** section. You can find your
 
 </details>
 
-This is interesting but it would be more useful to have a chart showing updates per device. We can do this by splitting the metric by `deviceId`. Select the `device-updates` metric in the `Log-based metrics` Namespace and then select the `Apply splitting` option to generate separate graphs for each device.
+Remember how we added `customDimension` with to the metrics, when we created them?
```
Comment (MaggieSalak) [inline 04-custom-metrics/README.md:180]: Technically we never use the term 'customDimension' in the code. It's a tag (key-value pair) with the device id as the value. In the documentation they also just call it tags: https://opentelemetry.io/docs/instrumentation/net/automatic/custom/#metrics

File: 04-custom-metrics/README.md [hunk 4]
```diff
@@ -183,7 +187,7 @@ This is interesting but it would be more useful to have a chart showing updates
 
 </details>
 
-Let's have a look at reported temperature values. Select the `temperature` metric, adjust the aggregation to Min, Max or Avg and change the graph type to `Bar chart`. We can now see the distribution of temperature measurements sent by our devices.
+Now, let's go and have a look at the second custom metrics we created, which is for the reported temperature values. Select the `temperature` metric, adjust the aggregation to Min, Max or Avg and change the graph type to `Bar chart`. We can now see the distribution of temperature measurements sent by our devices.
```
Comment (MaggieSalak) [inline 04-custom-metrics/README.md:190]:
```suggestion
Now, let's go and have a look at the second custom metrics we created, which tracks the reported temperature values. Select the `temperature` metric, adjust the aggregation to Min, Max or Avg and change the graph type to `Bar chart`. We can now see the distribution of temperature measurements sent by our devices.
```

File: 04-custom-metrics/README.md [hunk 5]
```diff
@@ -192,23 +196,27 @@ Let's have a look at reported temperature values. Select the `temperature` metri
 
 </details>
 
-In a previous section we covered the topic of creating a custom dashboard for our application. You can now pin the custom metrics to add them to your dashboard.
+You can also query your custom metrics like all the other logs you have.  For this go to the **Logs** section of the portal and query the `customMetrics` table to see more your custom metrics and details on them.
+
+Just as an example, looking at the `device-updates` metric entries. Can you find the device ID which you previously used to tag the metric updates?
```
Comment (MaggieSalak) [inline 04-custom-metrics/README.md:201]:
```suggestion
Just as an example, looking at the `device-updates` metric entries, can you find the device ID which you previously used to tag the metric updates?
```

File: 04-custom-metrics/README.md [hunk 6]
```diff
@@ -192,23 +196,27 @@ Let's have a look at reported temperature values. Select the `temperature` metri
...
-![Pin custom metric](./images/custom-metrics-dashboard.png)
+![Metric logs](./images/custom-metrics-logs.png)
 
-</details>
+</details>  
+
+In this section we added custom metrics to our .NET application and learned how to visualize them in Application Insights. OpenTelemetry allows you to add similar metrics to applications written in other languages. Although we won't go into the details here, this [page](https://opentelemetry.io/docs/instrumentation/java/manual/#metrics) provides useful details on how you can add custom metrics to our Java application. 🔎
 
-You can also query your custom metrics to access more details, such as custom dimensions, including the added tags. Go to the **Logs** section of the portal and query the `customMetrics` table to see more details of the custom metrics tracking successful and failed device updates.
+To learn about more features of the metrics view, as well as how to create and add these generated graphs into a custom dashboard, follow the next few chapters 😉.
```
Comment (MaggieSalak) [inline 04-custom-metrics/README.md:212]:
```suggestion
To learn about other features of the metrics view, as well as how to create and add these generated graphs into a custom dashboard, follow the next few chapters 😉.
```

File: 04-custom-metrics/README.md (hunk 3 above)
Comment (MahrRah) [reply 04-custom-metrics/README.md:180]: My bad, let me rephrase it :)

File: 04-custom-metrics/README.md [hunk 7]
```diff
@@ -174,7 +177,8 @@ Go to Application Insights and select the **Metrics** section. You can find your
 
 </details>
 
-This is interesting but it would be more useful to have a chart showing updates per device. We can do this by splitting the metric by `deviceId`. Select the `device-updates` metric in the `Log-based metrics` Namespace and then select the `Apply splitting` option to generate separate graphs for each device.
+Remember how we added `tags` to our metrics, when we created them?
+The graph created above is useful, but using our `customDimension` we could now go as far as to create a chart showing updates per device. We can do this by splitting the metric by `deviceId`. Select the `device-updates` metric in the `Log-based metrics` Namespace and then select the `Apply splitting` option to generate separate graphs for each device.
```
Comment (MaggieSalak) [inline 04-custom-metrics/README.md:181]:
```suggestion
The graph created above is useful, but using our `tags` we could now go as far as to create a chart showing updates per device. We can do this by splitting the metric by `deviceId`. Select the `device-updates` metric in the `Log-based metrics` Namespace and then select the `Apply splitting` option to generate separate graphs for each device.
```

File: 04-custom-metrics/README.md [hunk 8]
```diff
@@ -183,7 +187,7 @@ This is interesting but it would be more useful to have a chart showing updates
 
 </details>
 
-Let's have a look at reported temperature values. Select the `temperature` metric, adjust the aggregation to Min, Max or Avg and change the graph type to `Bar chart`. We can now see the distribution of temperature measurements sent by our devices.
+Now, let's go and have a look at the second custom metrics we created, which tracks the reported temperature values. Select the `temperature` metric, adjust the aggregation to Min, Max or Avg and change the graph type to `Bar chart`. We can now see the distribution of temperature measurements sent by our devices.
```
Comment (MaggieSalak) [inline 04-custom-metrics/README.md:190]:
```suggestion
Now, let's go and have a look at the second custom metric we created, which tracks the reported temperature values. Select the `temperature` metric, adjust the aggregation to Min, Max or Avg and change the graph type to `Bar chart`. We can now see the distribution of temperature measurements sent by our devices.
```

**Review by MaggieSalak — APPROVED** [review]
I just noticed 2 small things and added suggestions for them. Looks good otherwise:)
